src/functionalTest/java/uk/gov/hmcts/darts/testutil/ArmTestUtil.java

```diff
@@ -33,17 +37,23 @@ public void deleteBlobData(String containerName, String blobPathAndName) {
             Response<Boolean> response = blobClient.deleteIfExistsWithResponse(
                 DeleteSnapshotsOptionType.INCLUDE,
                 null,
-                Duration.of(DELETE_TIMEOUT, ChronoUnit.SECONDS),
+                Duration.of(deleteTimeoutInSeconds, ChronoUnit.SECONDS),
                 null
             );
 
-            log.info("deleteBlobData for container {}, Blob path {}, Returned status code {}", containerName, response.getStatusCode());
-            if (STATUS_CODE_202 != response.getStatusCode()) {
-                throw new AzureDeleteBlobException("Failed to delete from container because of http code: " + response.getStatusCode());
+            HttpStatus httpStatus = valueOf(response.getStatusCode());
+            if (httpStatus.is2xxSuccessful() || NOT_FOUND.equals(httpStatus)) {
+                log.info("deleteBlobData for containerName={}, blobPathAndName={}, httpStatus={}",
+                         containerName, blobPathAndName, httpStatus);
+            } else {
+                String message = String.format("Failed to delete from storage container=%s, blobId=%s, httpStatus=%s",
+                                               containerName, blobPathAndName, httpStatus);
+                throw new AzureDeleteBlobException(message);
```

Comment:
is this exception still tested?

Reply:
When the functional test was created we had no way to delete the objects being put in their. This provided this facility. It makes sense for the delete blobs to use similiar code. We may want to revisit this test as we may not want to using the arm blob storage this way, adding and removing test files